txcache/selectionTracker.go

```diff
@@ -189,3 +188,28 @@ func (st *selectionTracker) findNextBlock(previousHash []byte) *trackedBlock {
 
 	return nil
 }
+
+func (st *selectionTracker) createVirtualSelectionSession(
```

Comment:
Good naming across the function!

Reply:
Initially, I thought the algorithm would be as follows (pseudocode):

```
for each tracked block:
    create groups of breadcrumbs (sorted with respect to the chain of blocks), groups being indexed by sender address

for each group of breadcrumbs:
    create virtual account record
```

However, maybe your implementation is faster in our case. We'll learn during the final integration steps.